xtask/Cargo.toml

```diff
@@ -5,3 +5,4 @@ authors = ["The Yorick Developers"]
 edition = "2018"
 
 [dependencies]
+regex = "1"
```

Comment:
Is this dependency really just `1`?

Reply:
The changelog doesn't mention any new features used here.

Reply:
Yep!

Reply:
Wait, I'm wrong. We agreed to expand the full version numbers, as that's what `cargo upgrade` does. My bad.